e2e/tests/preferences.spec.ts

```
@@ -3,63 +3,73 @@
 import BrowserWrapper from "../infra/ui/browserWrapper";
 import GraphPage from "../logic/POM/graphPage";
 import PreferencesPage from "../logic/POM/preferencesPage";
+import ApiCalls from "../logic/api/apiCalls";
+import roles from '../config/user.json'
```

ESLint

[error] 'roles' is defined but never used.

(@typescript-eslint/no-unused-vars)